kivy/app.py

```diff
@@ -323,15 +323,12 @@ def on_resume(self):
 from kivy.event import EventDispatcher
 from kivy.lang import Builder
 from kivy.resources import resource_find
-from kivy.utils import platform as core_platform
+from kivy.utils import platform as platform
```

Comment:
Won't be ``from kivy.utils import platform`` enough?

Reply:
Thanks.  :)